ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -94,6 +94,183 @@ def test_basic_funnel_default_funnel_days(self):
         self.assertEqual(result[1]["count"], 1)
         self.assertEqual(len(result[1]["people"]), 1)
 
+    def test_basic_funnel_default_funnel_days_breakdown_event(self):
```

Comment:
perhaps these breakdown tests should go in `breakdown_cases.py` ?

Reply:
Very well